iridium-angular-client/projects/iridium-ui/src/environments/environment.ts

```diff
@@ -5,7 +5,7 @@ export const environment = {
     domain: 'http://localhost:8381/',
     redirectUri: 'http://localhost:4200/callback',
     successfulAuthDestination: '/dashboard',
-    clientId: 'YOUR_CLIENT_ID',
+    clientId: '6094a9e4f8e43a8441789a03d0230e7b',
```

Comment:
This was probably a mistake I would guess?

Reply:
This was, but I plan to make it a constant value, but configurable value to make the system easier to use.  Since it's a `clientID` the value is a publicly known a value and is not a secret.